tutorials/quickstarts/environment.yml

```diff
@@ -0,0 +1,29 @@
+name: azureml_py36
+dependencies:
+  # The python interpreter version.
+  # Currently Azure ML only supports 3.5.2 and later.
+- pip==20.2.4
+- python>=3.5.2,<3.8
+- nb_conda
+- boto3==1.15.18
+- matplotlib==2.1.0
+- numpy==1.18.5
+- cython
+- urllib3<1.24
+- scipy>=1.4.1,<=1.5.2
+- scikit-learn==0.22.1
+- pandas==0.25.1
+- py-xgboost<=0.90
+- conda-forge::fbprophet==0.5
+- holidays==0.9.11
+- pytorch::pytorch=1.4.0
+- cudatoolkit=10.1.243
+
+- pip:
+  # Required packages for AzureML execution, history, and data preparation.
+  - azureml-widgets~=1.22.0
+  - pytorch-transformers==1.0.0
+  - spacy==2.1.8
+  - https://aka.ms/automl-resources/packages/en_core_web_sm-2.1.0.tar.gz
```

Comment:
what is this? does not seem like a public pypi package

Reply:
Same as above, I can check with the owners of the how-to.